src/test/scala/scalaz/actors/ActorsSuite.scala

```diff
@@ -17,7 +17,7 @@ final class ActorsSuite extends RTS {
         case object Increase extends Message[Unit]
         case object Get      extends Message[Int]
 
-        val handler = new MessageHandler[Int, Nothing, Message] {
+        val handler = new Stateful[Int, Nothing, Message] {
```

Comment:
I'd use `Actor.stateful`. In fact, maybe best to make the `Stateful` class private so users have to use the `stateful` method.

Reply:
@jdegoes Trying to understand what you have in mind here. If we make `Stateful` private how would we provide `Actor.stateful` with an implmentation of `receive` for dealing with the states?

Reply:
No problem thanks for taking a look and giving me guidance on it. Will work on travis to pass and merge and move on to next thing.